joint_limits:


    JointA:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 0.27
      has_acceleration_limits: false
      max_acceleration: 0.2
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 255

    JointA:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 0.27
      has_acceleration_limits: false
      max_acceleration: 0.2
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 255